NewSpring/ApollosProject
# Adds Give and Serve to Actionbar

## DESCRIPTION

### What does this PR do, or why is it needed?
This PR adds give and serve to the action bar on the Connect screen.

### How do I test this PR?
Go to the Connect screen and see that the give and serve options are in the action bar and no longer in the action table. Make sure they function correctly.

## TODO


## REVIEW


**Manual QA**


---

> The purpose of PR Review is to _improve the quality of the software._
